try{
    console.log(num);
}catch (e){
    console.log("ERROR ASUUUUU");
}